react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css

```diff
@@ -0,0 +1,9 @@
+.webviz-labeled-container {
+    color: #555;
+    margin-top: 0.5vh;
+    margin-bottom: 0.5vh;
```

Comment:
`view height` sizes are no good practice here. Use `8px` or `16px` (multiple of 8)?

Reply:
`16px`